tests/e2e/main_test.go

```diff
@@ -63,6 +63,11 @@ func TestMain(m *testing.M) {
 		"",
 		`DCGM-exporter command line arguments. Example: -arguments="{-f=/etc/dcgm-exporter/default-counters.csv}"`)
 
+	flag.BoolVar(&testContext.noCleanup,
+		"no-cleanup",
+		false,
+		`Skip clean up after tests execution`)
+
```

Comment:
In what scenario we need this flag?

Reply:
@rohit-arora-dev Troubleshooting. E2E destroys all resources after the end of the test or cancelation. The flag allows to keep created resources for the investigation of failures.